### PR TITLE
Rebuild for aarch64 and ppc64le

### PR DESCRIPTION
**Pull request by xhochy**
Fixes #37

### CODE REVIEW

**Comment by xhochy** [comment]
@conda-forge/arm-arch @conda-forge/libcxx This is ready for review/merge.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,23 +9,25 @@ source:
...
+  number: 3
   skip: true  # [win and vc<14]
   skip: true  # [win]
   ignore_run_exports:
     - libstdcxx-ng
 
 requirements:
   build:
-    - cmake-binary
+    - cmake
```
[the remark this answers is not in the document]
Comment (xhochy) [reply recipe/meta.yaml]: I don't understand how this is introducing cycles. `cmake` doesn't seem to depend on it, or does it and I'm not seeing it?

[the remark this answers is not in the document]
Comment (isuruf) [reply recipe/meta.yaml]: On osx, it does

**Comment by xhochy** [comment]
@isuruf Addressed that in https://github.com/conda-forge/libcxx-feedstock/pull/41

**Pull request merged**